.env

```diff
@@ -0,0 +1,36 @@
+DATABASE_HOST=mariadb
+DATABASE_PORT=3306
+MYSQL_ROOT_PASSWORD=password
+DATABASE_NAME=specify
+# When running Specify 7 for the first time or during updates that 
+# require migrations, ensure that the MASTER_NAME and MASTER_PASSWORD 
+# are set to the root username and password. This will ensure proper 
+# execution of Django migrations during the initial setup. 
+# After launching Specify and verifying the update is complete, you can 
+# safely replace these credentials with the master SQL user name and password.
+MASTER_NAME=root
+MASTER_PASSWORD=password
```

Comment:
Did you want to use the default password for the database container? 
It is currently not `password`, so this will throw a rather unhelpful message for most users when building Specify 7:
`django.db.utils.OperationalError: (1045, "Access denied for user 'root'@'172.18.0.10' (using password: YES)")`

If the error is intentional and we do not want access to be granted until they modify this file, it would be helpful to either update the documentation (on either the Speciforum or on the Installation Instructions on GitHub... or both), or provide a more helpful error message if possible .

Reply:
The default password for the database container depends on what you configure it to be in the `.env` file. Did you try to deploy this for testing?

The `MYSQL_ROOT_PASSWORD` variable sets the root password for the container in the `.env` file. See my deployment without any modification:

```zsh
❯ docker exec -it 5c915398e224 /bin/bash                                                             at 06:54:10 PM
root@5c915398e224:/# mysql -uroot -ppassword
Welcome to the MariaDB monitor.  Commands end with ; or \g.
Your MariaDB connection id is 7
Server version: 10.11.5-MariaDB-1:10.11.5+maria~ubu2204 mariadb.org binary distribution
```

Reply:
Exactly, I switched it back to the original password, and I got the same error as you, in the other direction.
```
django.db.utils.OperationalError: (1045, "Access denied for user 'master'@'172.19.0.10' (using password: YES)")
```

So are you advocating for using `master` instead for backwards compatibility? As someone who switches between databases often, I prefer to use `root` (or another high level user) in testing/development so that I can run all migrations without needing to modify the configuration outside of changing the DB name.

Reply:
> The default password for the database container depends on what you configure it to be in the .env file. Did you try to deploy this for testing?

Interesting. Yes, you are correct. 
Turns out `docker container prune`, `docker image prune -a`, `docker volume prune -a`, and even `docker system prune -a` was not removing the old, dangling, or unused containers/images/volumes on my system.

I had to manually, individually remove the stopped containers,  images, and volumes to completely run Specify "from scratch". 
Even then, on the first time the containers were built Django threw an `OperationalError: Unknown MySQL server host 'mariadb' `. Stopping the containers and restarting them fixed this problem (most likely caused because the order the containers were build was "incorrect"?)

But yes, the `mariadb` does use the default password as specified in the `.env` file :partying_face: 

> So are you advocating for using master instead for backwards compatibility? As someone who switches between databases often, I prefer to use root

I also prefer to use `root` in such cases.

Reply:
@melton-jason If you want to go thermonuclear on docker, here is the command for that:

![Screenshot 2023-10-16 at 17 34 30](https://github.com/specify/specify7/assets/40512816/62caf096-6075-456f-af7a-a12d4d23656d)

it will delete volumes too

Reply:
Yes, thank you! 
I will be careful about wielding such power over Docker. :laughing: